src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+- **Elite hardsuit having a small chance to reflect energy weapons.**
+- **Juggernaut hardsuit increasing its damage resistance at the cost of increased energy usage.**
+- **Deathsquad hardsuit reflecting ballistic weapons (because screw you lol).**
+- **Deathsquad hardsuit being self-recharging (because screw you lol).**
+It's an admin-only spawn to kill everything and everyone. If any suit is going to be self-recharging, it's going to be this one.
+
+No other suit would ever get this ability, it goes against the very idea of this entire design doc the same way being able to swap out power cells would, you NEED to take a few minutes of downtime to charge your suit, using abilities and not using them is done with the knowledge that you are either delaying when you'll need that downtime or moving it closer it for special benefits.
+
+Generally, suit abilities should be able to replace an already existing tool, or be replaced by said tool to conserve charge, though certain suits doing unique things that fit their given job would be welcome as well (such as thermal regulators). It all gives an extra layer of depth and decisions to gameplay, and plenty more ways to screw things up on the fly if you're trying to micromanage suit charge and abilities. Your hardsuit is a tool to use to your advantage, just like every other piece of equipment you have. What you decide to do with it is up to you.
+
+## Several possible scenarios
+
+I would like to demonstrate the idea behind this proposal and the effects it can have on the game by giving four scenarios where hardsuit abilities interact with the rest of the game as a whole while adhering to the core game design pillars.
+
+- Imagine: A syndicate agent is using an engineering hardsuit for the welding protection ability, they have the plan to deconstruct into the armory to steal from it, so they also wear a pair of sunglasses. They have now attained the abilities of both welding goggles and sunglasses, but due to them leaving the welding goggle ability on after they were done using it, they run out of hardsuit power while fleeing security in space and are either apprehended or purchase a blood-red hardsuit/syndicate EVA suit from their uplink on the move. 
+
+- Imagine: An engineer is repairing a major spacing near cargo, but his suit runs out of power. He asks salvage if he can use one of their suit storages to charge, which they allow him to do so by opening the door. While the suit is charging, the salvager walks out of the room, leaving the engineer locked inside cargo to either call for the AI, hope the salvager returns, or repeatedly fail to open the door hoping the QM or a different salvager notices.
```

Comment:
Any LRP engineer would instantly get impatient and hack the door and you know this lmfao, but I get your point.